Translations/UltraPunishments_Italian.lang

```diff
@@ -1,47 +1,47 @@
 #SERVER: Server
-#PUNISH: Punish
-#OR: or
-#REPORTS: Reports
-#GUI_OVERVIEW_REPORTS_ACTION: Click to show opened reports
-#GUI_PLAYERVIEW_REPORTS_ACTIVEREPORTS_NONE: Player has no active reports
-#GUI_PLAYERVIEW_PUNISHMENTS_ACTIVE: Active Punishments
+#PUNISH: Punire
+#OR: o
+#REPORTS: Report
+#GUI_OVERVIEW_REPORTS_ACTION: Clicca per mostrare i report attivi
+#GUI_PLAYERVIEW_REPORTS_ACTIVEREPORTS_NONE: Il giocatore non ha report attivi
+#GUI_PLAYERVIEW_PUNISHMENTS_ACTIVE: Attiva la punizione
 #REPORT: Report
-#GUI_PLAYERVIEW_EXCLUDE_ACTION: Click to %1 punishments for this player
-#GUI_OVERVIEW_TEMPLATES_ACTION: Click to show templates
-#GUI_PLAYERVIEW_PUNISH_UNPUNISHABLE: Punishments for this player have been disabled.
-#TYPE_IN_MESSAGE: Type in a message
-#SET_A_DURATION: Click to set duration
-#PLAYERS: Players
-#REASON: Reason
+#GUI_PLAYERVIEW_EXCLUDE_ACTION: Fai click su %1 per punire questo giocatore
+#GUI_OVERVIEW_TEMPLATES_ACTION: Clicca per mostrare i template
+#GUI_PLAYERVIEW_PUNISH_UNPUNISHABLE: Le punizioni per questo giocatore sono disabilitate.
```

Comment:
Remove the dot at the end. It's misplaced in the original translation and not needed here